src/globals.d.ts

```diff
@@ -10,9 +10,10 @@ declare global {
   };
 
   interface Window {
-    React: any;
+    React: typeof import("react");
```

Comment:
Add import `import type React from "react";` above and change this to `React: typeof React;`